app/components/content-search/content-search.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+angular.module('bulbs.cms.contentSearch', [
+  'autocompleteBasic',
+  'bulbs.cms.site.config',
+  'contentServices.factory',
+  'lodash'
+])
+  .directive('contentSearch', [
```

Comment:
New `content-search` component. Allows searching, and optionally choosing, content.